Austral Summer Institute Course Wrap-up

Last month I had that oppurtunity to visit the University of ConcepciÃ³n, and teach a course at the [Austral Summer Institute](http://www.sur-austral.cl/asi/eng/descripcion.html) - "Physiological response of marine organisms to environmental change from a genomic perspective".
>>>
This is a five day course with AM lectures and hands-on bioinformatics modules in the afternoon. The learning objectives of this course include      
1) Understanding of fundamental physiological processes     
2) Acquisition of core computing skills    
3) Hands-on experience with genomic datasets    

 - via course website aka [GitHub wiki](https://github.com/sr320/austral/wiki)
 
 <img src="http://eagle.fish.washington.edu/cnidarian/skitch/IMG_0831_1A910442.jpg" alt="IMG_0831_1A910442.jpg"/>
 
 Mornings were physiology lectures with most of the day in the computer lab. We were very lucky to have Mac minis with the instructor's computer a wall-mounted  50 in HDTV.  The lab started with the Unix Shell thanks to [Software Carpentry lessons](https://github.com/sr320/austral/wiki/CCS_01). I trimmed the lesson a bit but think this was a good way to start as it gave the students something real text to reference. In the first day we made it through [blast at the commandline and SQLShare](http://nbviewer.ipython.org/github/sr320/austral/blob/master/modules/01-Piura-Annotation/00_Piura.ipynb). I used the command-line some so they could follow along but had prepared modules in IPython. The students were exposed to [IPython](http://ipython.org/) by day three using [SageMathCloud](https://cloud.sagemath.com/), though they did not use it regularly during the course. Day two was a carry-over from day one with an emphasis on gitting familar with the shell and transcriptome annotation.  We went on to [RNA-seq](https://github.com/sr320/ASI-rna-seq/blob/master/notebook/rna-seq_cgigas.md) with everyone also setting up their own [GitHub](https://github.com/) repository and learning how to operate with [GitHub for Mac](https://mac.github.com/). We did not get much more genomics done though I did show them a little [SNP](http://nbviewer.ipython.org/github/sr320/austral/blob/master/modules/04-Chile-SNP/00-Com-SNPs.ipynb) and [CpG prediction](http://nbviewer.ipython.org/github/sr320/austral/blob/master/modules/01-Piura-Annotation/02-Piura-CpG.ipynb) workflows. In the end It seems like everyone got at least something out of the course. As might be expected many students appreciated the web services that help make pretty pictures including [DAVID](http://david.abcc.ncifcrf.gov/home.jsp), [Revigo](http://revigo.irb.hr/), and [SQLShare](https://sqlshare.escience.washington.edu/sqlshare/).
 
 <img src="http://eagle.fish.washington.edu/cnidarian/skitch/iPhoto_1A91059E.png" alt="iPhoto_1A91059E.png"/>

If I were to do something like this again (which I hope I will be able to) a few things I might do is get more post-it note feedack and make an effort to get content to students ahead of time, so the have the oppurtunity to play with some of the tools. 